helpers.py

```diff
@@ -237,19 +237,24 @@ def get_sentences_speaker_mapping(word_speaker_mapping, spk_ts):
 
 
 def get_speaker_aware_transcript(sentences_speaker_mapping, f):
+    # Make sure mapping list isn't empty
+    if not sentences_speaker_mapping:
+        return
+
```

Comment:
I've never encountered this case before or had an issue that mentioned it, so I'd rather not add it to simplify code